week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,34 @@
+function onLoad() {
+
+    var butElement = document.getElementById('newDogPhoto');
+    butElement.addEventListener("click", function addDogPhotoToDOM() {
```

Comment:
It can be simple `function() {}`. Name is only required when we will refer to it somewhere else in the code.